modules/docs/arrow-docs/docs/docs/recursion/intro/README.md

```diff
@@ -185,12 +189,12 @@ The [Recursive]({{ '/docs/recursion/recursive' | relative_url }}) typeclass prov
 [Corecursive]({{ '/docs/recursion/recursive' | relative_url }}) typeclass provides `ana`, which are very
 similar to fold and unfold.
 
-```kotlin
+```kotlin:ank:silent
 typealias Algebra<F, A> = (Kind<F, A>) -> A     // fold
 typealias Coalgebra<F, A> = (A) -> Kind<F, A>   // unfold
 
-fun <F, A> Functor<F>.cata(f: Fix<F>, alg: Algebra<F, Eval<A>>): A
-fun <F, A> Functor<F>.ana(a: A, coalg: Coalgebra<F, A>): Fix<F>
+fun <F, A> Functor<F>.cata(f: Fix<F>, alg: Algebra<F, Eval<A>>): A = TODO()
```

Comment:
What do you mean by `TODO`?
Does that mean that it's a feature that is yet to come?

Reply:
not at all, it just to compile the doc because I cannot declare an abstract method at the top of the file

Reply:
ok thanks. Any reason why my compiler wouldn't see the `cata` and `ana` method of Functor? I can see `hylo` without problem

Reply:
those are in separate files/imports:
hylo -> import arrow.recursion.*
cata -> import arrow.recursion.extensions.fix.recursive.*
ana -> import arrow.recursion.extensions.fix.corecursive.*

Reply:
ok. Unfortunately, I cannot resolve those import. I'm using `0.8.2`.
I don't see any `extensions` subpackage under `arrow.recursion`. Is there an additional module to be imported?

Reply:
From arrow main page:

    compile "io.arrow-kt:arrow-recursion:$arrow_version"
    compile "io.arrow-kt:arrow-extensions-recursion:$arrow_version"

Reply:
oh that was the missing bit! Unfortunately that wasn't in `https://arrow-kt.io/docs/quickstart/libraries/` so I missed it. I think it's finally working thanks!

Reply:
you're welcome

Reply:
@cesartl that's on me, apologies!